CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 #### Changed
 
 - [#852](https://github.com/rails-sqlserver/activerecord-sqlserver-adapter/pull/852) Updated the column name matchers to accept database and owner names
+- [#858](https://github.com/rails-sqlserver/activerecord-sqlserver-adapter/pull/858) Allow table existence to be tested across database schemas.
```

Comment:
This one looks like a bug instead. Can you move it to `Fixed` section?

Reply:
Done